src/main/java/ca/warp7/frc2020/subsystems/ControlPanelSpinner.java

```diff
@@ -26,46 +31,62 @@
 
     private static ControlPanelSpinner instance;
 
+    //These are calibration vars, if you're getting too high or
+    //too low a RGB value then what it should be outputting you
+    //can use these to make adustments to the input.
+    private final int redMod = 0;
+    private final int greenMod = -61;
+    private final int blueMod = 0;
+    
     public static ControlPanelSpinner getInstance() {
         if (instance == null) instance = new ControlPanelSpinner();
         return instance;
     }
-
-    private CANSparkMax controlPanelMiniNeo = MotorControlHelper.createMasterSparkMAX(kControlPanelManipulatorID);
-    private ColorSensorV3 colorSensor = new ColorSensorV3(I2C.Port.kOnboard);
-
+    
+    @Override
+    public void periodic(){
+        SmartDashboard.putString("spinner", getCurrentSpinnerColor().toString());
+    }
+    
     /*
     Gets the current SpinnerColour that the
     ControlPanelSpinner's colour sensor is looking
     at.
     */
-    private SpinnerColour getCurrentSpinnerColor() {
+    public SpinnerColour getCurrentSpinnerColor(){
         final Color SENCED_COLOR = getCurrentColor();
-        final int R = (int) (255 * SENCED_COLOR.red);
-        final int G = (int) (255 * SENCED_COLOR.green);
-        final int B = (int) (255 * SENCED_COLOR.blue);
+        final int R = (int)(255 * SENCED_COLOR.red) + redMod;
+        final int G = (int)(255 * SENCED_COLOR.green) + greenMod;
+        final int B = (int)(255 * SENCED_COLOR.blue) + blueMod;
         final float[] HSB = new float[3];
         java.awt.Color.RGBtoHSB(R, G, B, HSB);
         return (convertHueToColour(HSB[0]));
     }
-
+    
     /*
     Given a HUE from 0-1, return one of the SpinnerColours based on
     a range of values.
     */
-    public SpinnerColour convertHueToColour(float hue) {
+    public SpinnerColour convertHueToColour(float hue){
```

Comment:
Documentation suggestion:  https://en.wikipedia.org/wiki/HSL_and_HSV
I think this is what we're measuring with hue,  https://en.wikipedia.org/wiki/File:HSL-HSV_hue_and_chroma.svg

Mapping to degrees so I'm in sync with the pictures,

red range 342 to 360 degrees 
yellow range 0 to 72
green range 144 to 216
blue range 230 to 252.  

I'd expect red to be around 0 degrees, yellow 60 degrees, green 120, and blue 240.  Everything is ballpark; the only thing that stands out is the yellow being from 0 to 72.  0 should be close to red.